packages/backend-common/api-report.md

```diff
@@ -560,6 +561,9 @@ export function loadBackendConfig(options: {
   logger: LoggerService;
   remote?: LoadConfigOptionsRemote;
   additionalConfigs?: AppConfig[];
+  additionalSchemas?: {
+    [context: string]: JsonObject;
+  };
```

Comment:
No need to add this right? or are you relying on the old backend system? Either way it'd be better to move to the new APIs for loading config: https://github.com/backstage/backstage/blob/fb50191fa3a289a990e4de1098525da5d6c9f6ae/packages/backend-app-api/src/services/implementations/config/rootConfigServiceFactory.ts#L45-L50

Reply:
> are you relying on the old backend system

No, the whole point of this PR is to enable us relying on the new backend system.

So you're right, let's remove it.